includes/block.php

```diff
@@ -333,12 +388,13 @@ function register_route() {
  */
 function rest_response( $request ) {
 	$attributes = array(
-		'customPostType' => $request->get_param( 'post_type' ) ? $request->get_param( 'post_type' ) : 'post,posts',
+		'customPostType' => $request->get_param( 'post_type' ) ? $request->get_param( 'post_type' ) : 'post',
```

Comment:
I'm not familiar enough with how `post,posts` is used, but it appears to still be used in the post type `<select>` input. Why change this back to `post` - entirely unrelated?

Reply:
I probably changed this because the post type is already being split in the JavaScript, with only the first part being passed in the request. But this isn't a necessary change, and I've reverted it :)